config/environments/development.rb

```diff
@@ -31,10 +33,22 @@
   # Store uploaded files on the local file system (see config/storage.yml for options).
   config.active_storage.service = :local
 
-  # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
 
-  config.action_mailer.perform_caching = false
+  config.action_mailer.default_url_options = { :host =>'localhost:3000' }
```

Comment:
Die Einstellung in Dev und Test für Mailer würde ich nicht auf SMTP einstellen. Man will ja bei der Entwicklung nicht falsche Mails senden. Dafür stellt mal lieber https://github.com/ryanb/letter_opener ein.